src/main/java/com/dev/cinema/annotation/CustomEmailValidator.java

```diff
@@ -0,0 +1,16 @@
+package com.dev.cinema.annotation;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+public class CustomEmailValidator implements ConstraintValidator<CustomEmailConstraint, String> {
+    @Override
+    public void initialize(CustomEmailConstraint constraintAnnotation) {
+
+    }
+
+    @Override
+    public boolean isValid(String email, ConstraintValidatorContext constraintValidatorContext) {
+        return email != null && email.matches("^(.+)@(.+)$");
```

Comment:
REGEX pattern has to be in constant checklist says

Reply:
done